api/model/src/main/java/org/projectnessie/model/CommitResponse.java

```diff
@@ -61,6 +61,24 @@ default Map<ContentKey, String> toAddedContentsMap() {
     return added.stream().collect(Collectors.toMap(AddedContent::getKey, AddedContent::contentId));
   }
 
+  /**
+   * If new content has been added to Nessie, updates the {@link Content#getId() content ID} of the
+   * given content with the generated content ID.
+   */
+  @SuppressWarnings("unchecked")
+  default <T extends Content> T contentWithId(ContentKey key, T content) {
+    List<AddedContent> addedContents = getAddedContents();
+    if (addedContents == null) {
+      return content;
```

Comment:
If the purpose of this method is for clients to obtain content IDs that have just been assigned, should we throw an exception here? If we get here that means the caller passed in a content that was either not part of the commit request, or was not "added". WDYT?

Reply:
well, but why are we adding this method? Like @adutra mentioned, it does not appear to be used ATM :thinking: